package.json

```diff
@@ -18,6 +18,10 @@
     "webrtc-adapter": "8.2.2"
   },
   "devDependencies": {
-    "eslint": "8.35.0"
+    "eslint": "8.35.0",
+    "eslint-plugin-html": "^7.1.0"
+  },
+  "scripts": {
+    "lint": "eslint html --ext .js,.html"
```

Comment:
What is the scope of `eslint` here?
Are we sure that the script is going to use the binary installed by npm (e.g. the one under `./node_modules`) and not a global one already installed in the environment?

Reply:
The locally installed version will be used (first) when using `npm run`